Components/TextUtilities/textutilities.css

```diff
@@ -0,0 +1,27 @@
+.heading-wrap{
+    display: flex;
+    flex-direction: column;
+    gap:10px
+}
+.heading-wrap{
+    border: 1px solid;
+    box-shadow: rgb(0 0 0 / 15%) 1.95px 1.95px 2.6px;
```

Comment:
You can declare variables for box-shadow in the component.css file since u are using the same property at multiple places.